### ISSUE TITLE
Feedback - Getting started - Hello world

### ISSUE THREAD

**Issue by LeReunionais** [tokio-rs/doc-push#73]
* Where to add dependencies
It's a tiny feedback but I think it would be nice to add in which file to add the dependencies:
Something like:

>Next, add the necessary dependencies in `Cargo.toml`:
>```
>[dependencies]
>tokio = "0.1"
>```

* Add a section at the end to run the example:
1. Start a tcp server: `nc -p 6142`
1. Run: `cargo run`
1. What to expect
```
About to create the stream and write to it...
created stream
wrote to stream; success=true
Stream has been created and written to.
```

**Comment by carllerche** [comment]
Looks good to me!

cc @rylev I think this would be applied to your section.

**Comment by mziter** [comment]
Running the following command per current docs gives me Connection refused error
`nc -l 6142`
I needed to run the following for the example to succeed
`nc -l -p 6142`

I would be happy to update once someone confirms that current docs are innacurate

**Comment by mziter** [comment]
PR in for requested change above: https://github.com/tokio-rs/website/pull/283

**Comment by mziter** [comment]
Closed PR. Looks like my issue was with the version of netcat I was using. What are people's thoughts on adding an info or warning section about possible version issues. I originally discovered this because I got an error about the connection being refused, which might put off users.

**Comment by carllerche** [comment]
Right, we probably should move away from using netcat here given how there are differences across platforms.

Does anyone have ideas on an alternative?
